Domain/Sia.Shared/Protocol/Response.cs

```diff
@@ -9,18 +9,10 @@ public class Response<T> : Response, IResponse<T>
     {
         public T Value { get; private set; }
 
-        public static new async Task<Response<T>> Create(HttpResponseMessage message)
+        public Response(HttpStatusCode statusCode, bool isSuccessCode, string content, T value) 
```

Comment:
Does it make sense to make this constructor internal given that we should only be instantiating these from the factory?